diagnostic.md

```diff
@@ -150,7 +182,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+Div One through three will float right to the top of the page, while div four cannot float left because of the clear, so will remian static. Same with divFive
```

Comment:
```.divOne``` will be in the top-right corner;
```.divTwo``` will be in the top-left corner, with .divThree immediately to its right;
```.divFour``` will be underneath ```.divTwo```, and ```.divFive``` will sit immediately to the right of ```.divFour``` (under ```.divThree```).